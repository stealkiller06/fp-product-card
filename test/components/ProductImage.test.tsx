import React from 'react'
import renderer from 'react-test-renderer'
import ProductCard, { ProductImage } from '../../src/components'
import { product2 } from '../data/product'

describe("ProductImage", ()=>{

    it("debe mostrar la imagen pasada por parametro",()=>{
        const wrapper = renderer.create(
            <ProductImage img={product2.img} />
        )

        expect(wrapper.toJSON()).toMatchSnapshot()
    })

    it("debe mostrar la imagen del context", ()=>{
        const wrapper = renderer.create(
            <ProductCard product={product2}>
                {()=>(
                <ProductImage/>

                )}
            </ProductCard>
        )
        expect(wrapper.toJSON()).toMatchSnapshot()
    })

    
})