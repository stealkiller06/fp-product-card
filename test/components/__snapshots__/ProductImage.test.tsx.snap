// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage debe mostrar la imagen del context 1`] = `
<div
  className="productCard "
>
  <img
    alt=""
    className="productImg "
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductImage debe mostrar la imagen pasada por parametro 1`] = `
<img
  alt=""
  className="productImg "
  src="./coffee-mug.png"
/>
`;
